### PR TITLE
Fixed #35147 -- Added backward incompatibility note about filtering against overflowing integers.

### PR DESCRIPTION
**Pull request by Darbados**

### CODE REVIEW

File: docs/releases/5.0.txt [hunk 1]
```diff
@@ -588,6 +588,11 @@ Miscellaneous
 
 * The minimum supported version of ``docutils`` is increased to 0.19.
 
+* Due to a commit meant to address a SQLite crash and a DDOS vector on
+  Postgres, non-wrapped arithmetic against integer fields might require
+  adjustments, like explicit wrapping with ``ExpressionWrapper``, see
+  (:ticket:`35147`).
```
Comment (felixxm) [inline docs/releases/5.0.txt:594]: We don't refer ticket number in a feature release notes. Maybe:
```suggestion
* Filtering querysets against overflowing integer values now immediately
  returns an empty queryset. As a consequence, you may need to use
  ``ExpressionWrapper`` to :ref:`explicitly wrap <using-f-with-annotations>`
  arithmetic against integer fields in such cases.
```

Comment (Darbados) [reply docs/releases/5.0.txt]: Thanks for the comment Felix. I guess the referrence to a ticket are only for Bugfixes. I rewrite the note per your suggestion.

File: docs/releases/5.0.txt [hunk 2]
```diff
@@ -588,6 +588,11 @@ Miscellaneous
 
 * The minimum supported version of ``docutils`` is increased to 0.19.
 
+* Filtering querysets against overflowing integer values now immediately
```
Comment (felixxm) [inline docs/releases/5.0.txt:591]: Maybe `always` :thinking: 
```suggestion
* Filtering querysets against overflowing integer values now always
```

Comment (Darbados) [reply docs/releases/5.0.txt]: Ugh, yeah, it is always, indeed.

**Comment by felixxm** [comment]
@Darbados Thanks :+1: Welcome aboard :boat:

**Pull request merged**